app/views/external_users/claims/misc_fees/advocates/_misc_fee_fields.html.haml

```diff
@@ -3,54 +3,45 @@
+  = f.govuk_fieldset legend: {} do
+    %span.govuk-heading-m
+      = t('.misc_fee_html')
 
     = link_to_remove_association f, wrapper_class: 'misc-fee-group', class: 'govuk-link govuk-!-display-none fx-numberedList-action' do
       = t('common.remove_html', context: t('.misc_fee'))
 
-    .form-group
-      .fee-type.js-typeahead{ class: error_class?(@error_presenter, "misc_fee_#{@misc_fee_count}_fee_type") ? 'form-group-error dropdown_field_with_errors' : '' }
-        = f.label :fee_type_id_input,
-          t('.fee_type_html', context: t('.misc_fee')),
-          { class: 'form-label-bold' }
-
-        = validation_error_message(@error_presenter, "misc_fee_#{@misc_fee_count}_fee_type")
-
-        = f.select :fee_type_id,
-          options_for_select(claim.eligible_misc_fee_type_options_for_select, f.object&.fee_type&.id),
-          { include_blank: ''.html_safe },
-          { class: 'form-control form-control-full js-misc-fee-type js-fee-type js-fee-calculator-fee-type typeahead fx-misc-fee-calculation',
-          id: "misc_fee_#{@misc_fee_count}_fee_type",
-          'aria-label': t('.fee_type') }
+    = f.govuk_select(:fee_type_id,
+      class: 'js-misc-fee-type js-fee-type js-fee-calculator-fee-type fx-misc-fee-calculation',
+      form_group: { classes: 'fee-type fx-autocomplete-wrapper cc-fee-type'},
+      label: { text: t('.fee_type_html', context: t('.misc_fee')), size: 's' },
+      options: { include_blank: '' }) do
+      = options_for_select(claim.eligible_misc_fee_type_options_for_select, f.object&.fee_type&.id)
 
     .form-group.fx-unused-materials-warning.js-hidden
       = render 'warnings/unused_material_over_3_hours'
 
-    = f.adp_text_field :quantity,
-                            label: t('.quantity_html', context: t('.misc_fee')),
-                            hint_text: t('.quantity_hint'),
-                            hide_hint: true,
-                            input_classes: 'quantity fee-quantity form-control-1-8 js-fee-quantity js-fee-calculator-quantity',
-                            input_type: 'number',
-                            value: fee.quantity,
-                            errors: @error_presenter
-
-    .js-unit-price-effectee
-      .calculated-unit-fee
-      = f.adp_text_field :rate,
-          label: t('.rate_html', context: t('.misc_fee')),
-          input_classes: 'rate fee-rate js-fee-calculator-rate form-input-denote__input form-control-1-4',
-          input_type: 'currency',
-          errors: @error_presenter,
-          input_readonly: f.object.price_calculated?,
-          value: number_with_precision(f.object.rate, precision: 2)
-
-    .js-fee-calculator-success
-      = f.hidden_field :price_calculated, value: f.object.price_calculated?
+    = f.govuk_number_field :quantity,
+      class: 'quantity fee-quantity js-fee-quantity js-fee-calculator-quantity',
+      form_group: { classes: 'quantity_wrapper' },
+      hint: { text: t('.quantity_hint'), hidden: true},
```

Comment:
Hidden hint??

Reply:
Just translating from existing adp to govuk formbuilder. Hint was hidden before.

Reply:
Done ✅